packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -53,8 +60,33 @@ const OdysseyThemeProvider = ({
     [odysseyTheme, themeOverride]
   );
 
+  const uniqueAlphabeticalId = useUniqueAlphabeticalId();
+
+  const cache = useMemo(
+    () =>
+      createCache({
+        ...(emotionRoot && { container: emotionRoot }),
+        key: uniqueAlphabeticalId,
+        prepend: true,
+        nonce: window.cspNonce,
+        speedy: false,
+      }),
+    [emotionRoot, uniqueAlphabeticalId]
+  );
+
+  if (withCache) {
+    return (
+      <CacheProvider value={cache}>
+        <MuiThemeProvider theme={customOdysseyTheme ?? odysseyTheme}>
+          <OdysseyDesignTokensContext.Provider value={odysseyTokens}>
+            {children}
+          </OdysseyDesignTokensContext.Provider>
+        </MuiThemeProvider>
+      </CacheProvider>
+    );
+  }
```

Comment:
I'm concerned about the added complexity here (if we otherwise discourage use of this provider directly). BUT if it's broken as-is (and *someone* is using it directly instead of OdysseyProvider), then it makes sense to make it functional.

Reply:
I agree with your thoughts on this. I don't have the context on why we need this provider (since `OdysseyProvider` already includes this one), but I wanted to make sure that the emotion cache worked and was available should someone use this provider directly.